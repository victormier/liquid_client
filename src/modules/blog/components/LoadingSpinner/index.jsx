import React from 'react';
import { Grid } from 'react-bootstrap';
import Spinner from 'components/common/Spinner';


const LoadingSpinner = () => (
  <Grid>
    <Spinner />
  </Grid>
);

export default LoadingSpinner;
