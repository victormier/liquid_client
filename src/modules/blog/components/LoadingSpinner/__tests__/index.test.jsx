import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import LoadingSpinner from '../index'


describe('component', () => {
  it('renders', () => {
    const component = shallow(
      <LoadingSpinner />
    )

    expect(shallowToJson(component)).toMatchSnapshot()
  })
})
