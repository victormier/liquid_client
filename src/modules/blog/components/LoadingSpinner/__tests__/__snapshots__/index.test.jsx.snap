// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <Spinner />
</Grid>
`;
